scalikejdbc-core/src/main/scala/scalikejdbc/ParameterBinderFactory.scala

```diff
@@ -143,14 +151,6 @@ private[scalikejdbc] object ParameterBinderFactoryMacro {
     import c.universe._
     val A = weakTypeTag[A].tpe
     val expr = A.toString match {
-      case "java.time.ZonedDateTime" | "java.time.OffsetDateTime" =>
-        q"scalikejdbc.ParameterBinderFactory[$A] { v => (ps, idx) => ps.setTimestamp(idx, java.sql.Timestamp.from(v.toInstant)) }"
-      case "java.time.LocalDateTime" =>
-        q"scalikejdbc.ParameterBinderFactory[$A] { v => (ps, idx) => ps.setTimestamp(idx, java.sql.Timestamp.valueOf(v)) }"
-      case "java.time.LocalDate" =>
-        q"scalikejdbc.ParameterBinderFactory[$A] { v => (ps, idx) => ps.setDate(idx, java.sql.Date.valueOf(v)) }"
-      case "java.time.LocalTime" =>
-        q"scalikejdbc.ParameterBinderFactory[$A] { v => (ps, idx) => ps.setTime(idx, java.sql.Time.valueOf(v)) }"
```

Comment:
Is there any reason does not remove `private[scalikejdbc] object ParameterBinderFactoryMacro` and `LowPriorityImplicitsParameterBinderFactory0` ?